frontend/website/static/js/leaderboard.js

```diff
@@ -0,0 +1,47 @@
+function renderParticipant(participant, rank) {
```

Comment:
I think we should come up with some guidelines as a team around how large something has to be before we want to try to get in reason instead of JS.

Like this feels like JS is the right thing, but if it was 200lines of logic feels like we should come up with some way to build it in reason.

Anyway, this is fine it's <50 loc

Reply:
Maybe this is something that gets fixed for us automatically when we move to a more dynamic website with client-side rendered react

Reply:
Yeah the problem is anything dynamic gets tricky. I think we should move away from this server side rendered model soon.